test/e2e/util.go

```diff
@@ -105,6 +105,83 @@ func providerIs(providers ...string) bool {
 
 type podCondition func(pod *api.Pod) (bool, error)
 
+// podReady returns whether pod has a condition of Ready in the status True.
```

Comment:
I think it's actually correct. To explain: pod has an array of conditions. One of the conditions it can have is the readiness condition (called "Ready"). This condition can have a status of True or False. We want the ready condition to have status True.